src/components/academy/NavigationBar.tsx

```diff
@@ -33,6 +46,12 @@ const NavigationBar: React.SFC<NavigationBarProps> = props => (
       >
         <Icon icon={IconNames.LIGHTBULB} />
         <div className="navbar-button-text hidden-xs">Quests</div>
+        <NotificationBadge
+          notifications={props.notifications.filter(
+            n => n.type !== 'submitted' && n.assessment_type && n.assessment_type === 'Sidequest'
```

Comment:
This is confusing, refactor to differentiate whether a notification belongs to an assessment or submission